### PR TITLE
ODC-7425: Add serverless function icon in Add page group header

### PR DESCRIPTION
**Pull request by lokanandaprabhu**
**Story**: https://issues.redhat.com/browse/ODC-7425

**Screenshots:** 

Screenshot 2023-11-17 at 11 27 18 AM
Screenshot 2023-11-17 at 11 26 53 AM

### CODE REVIEW

**Review by Lucifergene — COMMENTED** [review]
Verified. Looks good :)
image

/lgtm

File: frontend/packages/dev-console/src/components/add/AddCard.tsx [hunk 1]
```diff
@@ -9,14 +12,36 @@ type AddCardProps = {
...
+        </span>
+      );
+    }
+    return null;
+  };
   return items?.length > 0 ? (
     <Card key={title} className="odc-add-card" data-test={`card ${id}`}>
       {!isTitleFromItem && (
-        <Title size="lg" headingLevel="h2" className="odc-add-card__title" data-test="title">
+        <Title size="lg" headingLevel="h2" className="odc-add-card__title" data-test="title-1">
```
Comment (jerolimov) [inline frontend/packages/dev-console/src/components/add/AddCard.tsx:43]: Is this `-1` needed? :smirk: 

```suggestion
        <Title size="lg" headingLevel="h2" className="odc-add-card__title" data-test="title">
```

**Comment by lokanandaprabhu** [comment]
Hi @jerolimov ,

I have updated the PR. PTAL.

**Review by jerolimov — APPROVED** [review]
reapply lgtm

/lgtm
/approve

**Comment by sanketpathak** [comment]
Verified on cluster-bot with admin and non-admin users
Screenshot 2023-11-20 at 11 53 57 PM

/label qe-approved

**Comment by lokanandaprabhu** [comment]
Propage labels from the epic:

/label docs-approved
/label px-approved

**Pull request merged**